### PR TITLE
Add Combined dependency manager file

### PR DESCRIPTION
**Pull request by ATamblingPoder**
This file will enable us to remove all the different distro_install.sh scripts for ease of use.
Partly fixes both #15 and #16 
Suggestions welcome! :)
Regards

### CODE REVIEW

File: check_deps.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+build_liboqs() {
+	git clone -b main https://github.com/open-quantum-safe/liboqs.git
+	cd liboqs
+	mkdir -p build && cd build
+	cmake -GNinja .. -DBUILD_SHARED_LIBS=ON
+	ninja
+	sudo ninja install
+	cd ..
+	rm -rf liboqs
```
Comment (Anish-M-code) [inline check_deps.sh]: @ATamblingPoder I guess we don't need to remove liboqs folder at present , Pqcrypt needs structural changes and currently I am planning to use cloned liboqs folder to derive unique version of pqcrypt for issue #11

**Review by Anish-M-code — CHANGES_REQUESTED** [review]

**Comment by Anish-M-code** [comment]
I feel it would be better if we could combine run.sh as well in a similar way making pqcrypt easy to use.

**Comment by ATamblingPoder** [comment]
> I feel it would be better if we could combine run.sh as well in a similar way making pqcrypt easy to use.

Yeah I think that'll be much better! Meanwhile I've removed the rm line and I'll start work on this soon
Also it's my first time participating in Hacktoberfest 😄

**Comment by Anish-M-code** [comment]
@ATamblingPoder Thanks for contributing !

**Pull request merged**